### ISSUE TITLE
microk8s enable gpu: Error: INSTALLATION FAILED: failed to download "nvidia/gpu-operator" at version "v22.9.0"

### ISSUE THREAD

**Issue by McToel** [canonical/microk8s#3686]
#### Summary
I have a fresh installation of microk8s on a server where I have used the gpu addon before.
When I do:
`microk8s enable gpu`
I get the following:
```
Infer repository core for addon gpu
Enabling NVIDIA GPU
Addon core/dns is already enabled
Addon core/helm3 is already enabled
Checking if NVIDIA driver is already installed
GPU 0: NVIDIA GeForce RTX 3080 (UUID: GPU-95a02308-065c-64ee-26a8-e1ec8e4cfe60)
Using host GPU driver
"nvidia" already exists with the same configuration, skipping
Error: INSTALLATION FAILED: failed to download "nvidia/gpu-operator" at version "v22.9.0"
NVIDIA is enabled
```
I guess this should not happen. Doing `microk8s status` tells me, that the gpu addon is disabled.

Are there any docs on how to troubleshoot the gpu addon installation?

#### Introspection Report

[inspection-report-20230123_215513.tar.gz](https://github.com/canonical/microk8s/files/10483573/inspection-report-20230123_215513.tar.gz)

**Comment by neoaggelos** [comment]
Hi @McToel,

Can you check if the following helps with your issue?

```
microk8s helm repo update nvidia
microk8s enable gpu
```

**Comment by McToel** [comment]
Yes, that did the trick. Thanks a lot. Is there any place in the docs where this would fit in?

**Comment by neoaggelos** [comment]
This has actually been fixed in newer versions https://github.com/canonical/microk8s-core-addons/pull/128, but not backported to 1.25. I suppose a note in the docs could help. Thanks for reporting this!

**Comment by neoaggelos** [comment]
Also added a note in the [GPU addon documentation page](https://microk8s.io/docs/addon-gpu). Let us know if we can close the issue. Thanks again @McToel!
